app/javascript/components/__tests__/IcebreakerQuestion.test.js

```diff
@@ -24,15 +24,15 @@ describe('IcebreakerQuestion', () => {
   test('checking components for crashing', () => {
     render(
       <MemoryRouter>
-        <IcebreakerQuestion data={data} setData={setData} service={mockService} />
+        <IcebreakerQuestion data={data} setData={setData} saveDataToDb={() => {}} steps={[]} service={mockService} draft={false} />
```

Comment:
Between here and line 35, investigate whether result render blocks can be stored in a single variable to reference in multiple tests.